seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -453,7 +490,7 @@
     variant['genotypes'].update({
         'I000015_na20885': {
             'ab': 0.631, 'ad': None, 'gq': 99, 'sampleId': 'NA20885', 'numAlt': 1, 'dp': 50, 'pl': None,
-            'cn': 2, 'end': None, 'start': None, 'numExon': None, 'defragged': None, 'qs': None, 'sampleType': 'WES',
+            'cn': 2, 'end': None, 'start': None, 'numExon': None, 'defragged': None, 'qs': None, 'gqSv': None, 'sampleType': 'WES',
```

Comment:
@ShifaSZ this is what I am taking about - why on earth is this being created/ returned if it is never populated? The response config should be totally different from the query config

Reply:
Yes. I know. The `gqSv` is not a field that we want.